src/run_stream_client.py

```
@@ -0,0 +1,168 @@
+
+    base_url = args.base_url.rstrip("/")
+    if args.agent_id:
+        url = f"{base_url}/{args.agent_id}/stream"
+    else:
+        url = f"{base_url}/stream"
+
+    print(f"\n[요청] POST {url}")
+    print(f"payload: {json.dumps(payload, ensure_ascii=False)}\n")
+
+    # 3) SSE 스트림 열기
+    async with httpx.AsyncClient(timeout=None) as client:
```

Ruff (0.14.14)

Probable use of `httpx` call with timeout set to `None`

(S113)